### DISCUSSION TITLE
RTC for time tracking.

### DISCUSSION THREAD

**Discussion by ashish8284** [omersiar/esp-rfid#614] [Q&A]
Hi,
Is it possible to integrate I2C RTC for  time keeping ???

**Comment by matjack1** [comment]
Hello @ashish8284 !

It would be nice to have that option. There is already an issue open here: https://github.com/esprfid/esp-rfid/issues/508 if you want to help I'm happy to guide you and review the code, but I don't have time to implement that myself.
